buildlib/azure-pipelines-pr.yml

```diff
@@ -144,8 +144,15 @@ stages:
               set -eE
               gcc --version
               ./contrib/configure-release
-              ./contrib/buildrpm.sh -s -t -b
-            displayName: Build tarball
+              stdbuf -e0 -o0 ./contrib/buildrpm.sh -s -t -b |& tee rpmbuild.log
+              pattern='^warning: '
```

Comment:
i dont think pattern ```^warning:``` is correct, it searches from col #1 of str, while warning: in log file prefixed with file:line

Reply:
the log file does not have timestamp prefix (only AZP adds it later)
i any case, i've tested that when SPEC is wrong, it finds it: https://github.com/openucx/ucx/runs/845557002